docs/CHANGELOG.md

```diff
@@ -5,11 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Fixed
+- Rework and complete the tracker stream pre-selection module to fix it
 
 ## [4.3.4] - 2024-10-03
 ### Added
 - Create custom DjangoQL lookup field for Flaw.components (OSIDB-3479)
 - Collect NIST CVSSv4 in NVD collector (OSIDB-2300)
+- Introduce moderate tracker streams pre-selection (OSIDB-3346)
```

Comment:
This should go to the Unreleased section

Reply:
Oh...I see I rebased only one of the two records there :facepalm: